sdk/search/search-documents/src/searchClient.ts

```diff
@@ -575,6 +575,16 @@ export class SearchClient<T> {
     }
   }
 
+  /**
+   * Delete a set of documents by their primary key.
```

Comment:
nit: same thought, "Delete a set of documents."

Reply:
Done.